tfjs-backend-webgpu/src/index.ts

```diff
@@ -42,6 +42,10 @@ registerBackend('webgpu', async () => {
 
   if (supportTimeQuery) {
     deviceDescriptor = {extensions: ['timestamp-query' as const ]};
+  } else {
+    console.warn(`The webgpu backend only supports to asynchronously read ` +
+        `data from GPU side, but the device doesn't support timestamp-query ` +
+        `extension, you will get incorrect time if you profile operators.`);
```

Comment:
How about giving a warning like below:
```
This device doesn't support timestamp-query extension. Zero will shown for the kernel time when profiling mode is enabled. Using performance.now is not workable for webgpu since it doesn't support synchronously to read data from GPU.

Reply:
I am not sure what's more appropriate to throw this warning. Does someone else have comments?